### DISCUSSION TITLE
Issue with subdomain.

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#2392] [Help] [answered]
### Summary

My domain is `disland.uk.to` (on cloudflare)

Now the issue is, Vercel asks me to put the verification record in `_vercel.uk.to`. Which I cannot as I only have control over `disland.uk.to` (Basically I have control over `*.disland.uk.to`).

**Comment by ghost** [comment]
**It is possible for me to add a `_vercel.disland.uk.to` record, But vercel does not check that record's existance.**

**Comment by smaeda-ks** [comment]
seems like `uk.to` isn't recognized as a valid eTLD: https://github.com/publicsuffix/list/blob/master/public_suffix_list.dat

We will see what we can do.

**Reply by ghost** [reply]
I forgot that github has a way to explicitly reply to comments in discussions. (Re-added the stuff here)

Cloudflare and other most websites have recognized it as a valid eTLD. (They actually verify it using the records from the subdomain as a example subdomain1.subdomain2.uk.to instead of subdomain1.uk.to which vercel uses _vercel.uk.to instead of _vercel.subdomain2.uk.to which allows this type of domains to work.)

**Reply by smaeda-ks** [reply, marked as answer]
@blindful are you using [Free DNS](https://freedns.afraid.org/)?
https://freedns.afraid.org/domain/registry/ I see `uk.to` as their shared domain there. Unfortunately, because this isn't technically a legit eTLD (e.g., you can't buy `foo.uk.to` with any commercial domain registrar), it's hard to support this on our platform today.

**Reply by ghost** [reply]
Okay! I guess I'll go use a different platform instead.
